native/spotbugs-exclude.xml

```diff
@@ -0,0 +1,55 @@
+<!--
```

Comment:
The spotbugs rules here are from http module. I don't think we require any excludes. Let's verify and remove.

Reply:
Verified and removed in https://github.com/ballerina-platform/module-ballerina-observe/pull/1/commits/ed8df9f5f8ca5121b0dd0a3a15cb43f5ed96749a